src/Lean/Data/JsonRpc.lean

```diff
@@ -27,7 +27,7 @@ instance : OfNat RequestID n := ⟨RequestID.num n⟩
 
 instance : ToString RequestID where
   toString
-  | RequestID.str s => s!"\"s\""
+  | RequestID.str s => s
```

Comment:
```suggestion
  | RequestID.str s => s!"\"{s}\""
```
I think the quotation marks are significant here.  Otherwise you can't distinguish `1` and `1`.